bin/rialto/node/src/chain_spec.rs

```diff
@@ -191,3 +193,30 @@ fn load_kovan_bridge_config() -> BridgeKovanConfig {
 		initial_validators: rialto_runtime::kovan::genesis_validators(),
 	}
 }
+
+// We use this to get the account on Rialto (target) which is derived from Millau's (source) Dave
+// account. We do this so we can fund the derived account on Rialto at Genesis to it can pay
+// transaction fees. The reason we can use Rialto's Dave for the `millau_account_id` is because
+// they're both derived from the same seed phrase.
+//
+// In case you need it, the derived account ID is: 5Hg7WQyk8C1FmPzxY3xSjR7S6zZZC5sAL35vMr6NpW17jBhQ
+//
+// Note that this should only be used for testing.
+fn derived_dave_account_id() -> AccountId {
+	use sp_runtime::traits::Convert;
+	let millau_account_id = get_account_id_from_seed::<sr25519::Public>("Dave");
+	let encoded_id = bp_runtime::derive_account_id(*b"mlau", bp_runtime::SourceAccount::Account(millau_account_id));
+	bp_rialto::AccountIdConverter::convert(encoded_id)
```

Comment:
Can we put this to `bp_rialto` as a helper function? I imagine it's going to be quick useful for any tools that build on top of rialto-runtime, and you had to repeat it twice already in this file.